docs/src/main/sphinx/release/release-406.md

```diff
@@ -33,9 +35,11 @@
   filters. ({issue}`15552`)
 * Fix potential query failure when writing to Parquet from a table with an
   `INTEGER` range on a `BIGINT` column. ({issue}`15496`)
+* Fix query failure due to missing null counts in Parquet column indexes. ({issue}`15706`)
 
 ## Hive connector
 
+* Add support for table redirections to the Hudi connector. ({issue}`14750`)
```

Comment:
Check how we named this in the past and in the docs .. maybe

to catalogs using the Hudi connector.

Reply:
The sentence is the same as the past release notes. 
> Add support for table redirections to the Hive connector. ([#11356](https://github.com/trinodb/trino/issues/11356))
https://trino.io/docs/current/release/release-376.html#iceberg-connector